src/Agent.Plugins/PipelineCache/PipelineCacheTaskPluginBase.cs

```diff
@@ -91,13 +91,13 @@ public async virtual Task RunAsync(AgentTaskPluginExecutionContext context, Canc
                 context.Warning(OldKeyFormatMessage);
             }
 
-            context.Output($"Resolving key: {string.Join("|", keySegments)}");
+            context.Output($"Resolving key:");
```

Comment:
Can you share some before/after output?

I want to make sure folks can still see both the resolved and unresolved key segments

Reply:
Before the change:
![image](https://user-images.githubusercontent.com/43892780/66431268-8aa43f00-e9d0-11e9-90b6-dd02761db437.png)


Intermediate: 
![image](https://user-images.githubusercontent.com/43892780/66430687-73b11d00-e9cf-11e9-93c9-2fdad8bdd731.png)

After the change:
![image](https://user-images.githubusercontent.com/43892780/66430821-b1ae4100-e9cf-11e9-92ef-bbe7b077f51f.png)

Basically, in the FingerPrintCreator, we are displaying both the file and it's hash, just that, it will be wrt container, if the job is running inside the container.

Reply:
Removed this line, because there would be a discrepancy in display, (intermediate pic).

Reply:
"I want to make sure folks can still see both the resolved and unresolved key segments" - this will break. If we are ok with the intermediate display?

Reply:
As discussed, we are keeping both resolved and unresolved output. Reverting this code change.

Reply:
ok those all look good - "After" is best